spinalcordtoolbox/scripts/sct_run_batch.py

```diff
@@ -123,6 +123,11 @@ def get_parser():
                         help='Optional space separated list of subjects to exclude. Only process '
                         'a subject if they are not on this list. Inclusions are processed before exclusions. '
                         'Cannot be used with either `exclude`.', nargs='+')
+    parser.add_argument('-ignore-ses', action='store_true',
```

Comment:
I try to write flags and conditionals in a "positive" direction, to avoid the inevitable double negations that come up (like "not ignore"). How about something like `-only-sub`? Then again, maybe users that run into problems with `ses` folders will scan the options for something about `ses` rather than something about `sub`.

Sorry for the bike-shedding :sweat: